plugins/ruby-foreman-discovery/debian/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-foreman-discovery (4.1.2) stable; urgency=low
```

Comment:
Note that you'll need to append -1 to the version number as per #867 until it's merged.